test/e2e/ipsec_test.go

```diff
@@ -42,6 +42,7 @@ func TestIPSec(t *testing.T) {
 		t.Fatalf("Error when setting up test: %v", err)
 	}
 	defer teardownTest(t, data)
+	skipIfMulticastEnabled(t, data)
```

Comment:
ditto

Reply:
Regarding the IPSec test part, do you mean to do config mutation as wireguard test?

Reply:
No, my thought is to keep the two consistent, either both skip the case that "multicast.enabled=false", or use mutate to enforce they are run.

Reply:
Updated.